cmd/gateway/s3x/s3.proto

```diff
@@ -101,10 +94,10 @@ message ObjectInfo {
 
 // ObjectPartInfo contains information an individual object client
 message ObjectPartInfo {
-    // this should be an "int" type in Golang
+    // convertable to "int" type in minio.PartInfo
     int64 number = 1;
     string name = 2;
-    string etag = 3;
+    //string etag = 3; also use dataHash
```

Comment:
If this field is no longer going to be used, we should decrease the field number. With protocol buffers you always want to make sure that you increment the field numbers `1, 2, 3, 4, 5, etc...` as it requires the fewest bytes to encode the information for the first `1->15` fields.

Reply:
ok